README.md

```diff
@@ -11,9 +11,15 @@ Requirements
 - [SFML 2.5](http://www.sfml-dev.org/download.php)
 - [CSFML 2.5](http://www.sfml-dev.org/download/csfml/)
 
-Usage
+Documentation
 =====
-You can find documentation about `rust-sfml` into the [dedicated wiki](https://github.com/jeremyletang/rust-sfml/wiki)
+The SFML official API (C++) is available here: https://docs.rs/sfml/0.14.0/sfml/
```

Comment:
This isn't the official C++ API. This is the actual Rust API of rust-sfml.
Also, we don't need to link to a specific version.

I would personally write this as

> The API documentation is available at: https://docs.rs/sfml/

Reply:
Yeah... I am very sorry